app/controllers/applications_controller.rb

```diff
@@ -50,21 +50,30 @@ def update
 
     def admin_show
         @application = Application.find(params[:id])
-
     end
 
     def admin_update
         @application = Application.find(params[:id])
-        if params[:search].present?
-            @pets = Pet.search(params[:search])
+        if params[:pet_id] != nil
+            @pets = Pet.find(params[:pet_id])
+            @application_pets = ApplicationPet.find_by(pet_id: params[:pet_id], application_id: params[:id])
```

Comment:
I can't believe how long we were stuck on this one, SO happy we figured it out! TIL about param passing